vendor/holidays/lib/generated_definitions/sa.rb

```diff
@@ -15,7 +15,8 @@ def self.holidays_by_month
                 7 => [{:mday => 19, :year_ranges => { :limited => [2021] },:name => "Day of Arafah", :regions => [:sa]},
             {:mday => 20, :year_ranges => { :limited => [2021] },:name => "Day 1 of Eid Al Adha", :regions => [:sa]},
             {:mday => 21, :year_ranges => { :limited => [2021] },:name => "Day 2 of Eid Al Adha", :regions => [:sa]},
-            {:mday => 22, :year_ranges => { :limited => [2021] },:name => "Day 3 of Eid Al Adha", :regions => [:sa]}]
+            {:mday => 22, :year_ranges => { :limited => [2021] },:name => "Day 3 of Eid Al Adha", :regions => [:sa]}],
+      9 => [{:mday => 23, :name => "Saudi National Day", :regions => [:sa]}]
```

Comment:
Just want to confirm this wasn't added by hand right cause the indentation feels off to me.

Reply:
Nope, the rake task did it like that 🤷‍♂️